### ISSUE TITLE
Login not working

### ISSUE THREAD

**Issue by eni9889** [sheeehy/Geo-Git-v2#4]
I am trying to sign in to the live site and it's not working. Is the sign in down?

**Comment by sheeehy** [comment]
Yeah I haven't maintained it - think the server was hosted on Heroku, must be down.
